bin/nbb

```diff
@@ -344,6 +339,24 @@ find_centos_7_ami_ids() {
   done
 }
 
+shutdown_instances() {
+  aws ec2 describe-instances --filters "Name=instance-state-name,Values=running" | jq -c -r '.Reservations[] .Instances[] .InstanceId' > instance_list
+  echo "You are about to terminate the following instances:"
+  cat instance_list
+  read -r -p "Are you sure? [y/N] " response
+  if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]
+  then
+    IFS=$'\n'  # make newlines the separator
+    for i in `cat ./instance_list`; do
+      echo "Shutting down $i"
+      aws ec2 terminate-instances --instance-ids $i
```

Comment:
If that's the problem @7hunderbird is trying to solve, then this is entirely the wrong way to go about it, in my understanding.

The old way logs into CF, asks the bosh master to delete all the cloud foundry instances (runner/0, etc.)  -- which used to work and stopped working  Why doesn't seem to have been investigated.

This way uses a command line to delete three instances ( `--filters "Name=tag:Name,Values=bastion,nat,bosh/0"` ) two of which are _managed by terraform_ and doesn't cover any of the cloud foundry instances.  There will still be instances that will stick around if someone deploys cloudfoundry to them.